source/mir/math/stat.d

```diff
@@ -189,12 +189,12 @@ struct MeanAccumulator(T, Summation summation)
     ///
     size_t count;
     ///
-    Summator!(T, summation) sumAccumulator;
+    Summator!(T, summation) summator;
 
     ///
     F mean(F = T)() @property
     {
-        return cast(F) sumAccumulator.sum / cast(F) count;
+        return cast(F) summator.sum / cast(F) count;
```

Comment:
personally I like `summator`. If we have a special name for another kind of accumulator, we can use it though.

Reply:
@9il  I had decided that if I use `VarianceAccumulator varianceAccumulator` then I should use `Summator summator`. It is more consistent.